backend/Tim.Backend/DataProviders/Clients/KustoQueryClient.cs

```diff
@@ -165,7 +161,7 @@ private async Task<IEnumerator<ProgressiveDataSetFrame>> ExecuteQueryAsync(strin
             }
 
             m_logger.Information($"Executing query using Kusto's ExecuteQueryV2Async. Client Request Id: {clientRequestProperties.ClientRequestId} ", "KustoQueryClient-ExecuteQueryAsync");
-            var dataSet = await m_client.ExecuteQueryV2Async(databaseName, query, clientRequestProperties);
+            var dataSet = await m_client.ExecuteQueryV2Async(query.Cluster, query.Database, query.Query, clientRequestProperties);
```

Comment:
looking at this query.Query it really makes me want to name the property QueryText instead

Reply:
yeah, that's fair
I'll keep it as query in the json for the frontend though